src/main/scala/org/mbari/vars/annotation/dao/jpa/ImagedMomentImpl.scala

```diff
@@ -115,6 +115,23 @@ import scala.collection.mutable
     new NamedNativeQuery(
       name = "ImagedMoment.countByVideoReferenceUUID",
       query = "SELECT COUNT(*) FROM imaged_moments WHERE video_reference_uuid = ?1"
+    ),
+    new NamedNativeQuery(
+      name = "ImagedMoment.countAll",
+      query = "SELECT COUNT(*) FROM imaged_moments"
+    ),
+    new NamedNativeQuery(
+      name = "ImagedMoment.countWithImages",
+      query = "SELECT COUNT(*) FROM imaged_moments i " + 
```

Comment:
Change to `COUNT(DISTINCT i.uuid)`. Other wise you get a count of sum(imagedMoment * it's image references)

Reply:
For example: as written returns 1,154,916. Using distinct returns 606,273 (i.e. There's a png and jpg on each imagemoment)

Reply:
Done